docs/source/features/sheet_management/dataframe.rst

```diff
@@ -4,3 +4,123 @@
 ###########
  DataFrame
 ###########
+
+:class:`~graviti.dataframe.frame.DataFrame` is an integrated data structure
+with an easy-to-use API for simplifying data processing in Dataset.
+A Graviti DataFrame contains the columns Protex schema defines and the data resides in the rows and columns.
+
+************************
+ Initialize a DataFrame
+************************
+
+Initialize a Dataframe by a List:
+
+.. code:: python
+
+   from graviti import DataFrame
+   data = [
+       {"filename": "a.jpg"},
+       {"filename": "b.jpg"},
+       {"filename": "c.jpg"},
+   ]
+   df = DataFrame(data)
+
+Initialize a DataFrame with Multi-level column names:
+
+.. code:: python
+
+   from graviti import DataFrame
+   data = [
+       {"attribute": {"weather": "sunny", "color": "red"}},
+       {"attribute": {"weather": "rainy", "color": "black"}},
+       {"attribute": {"weather": "sunny", "color": "white"}},
+   ]
+   df = DataFrame(data)
+
+Initialize a DataFrame with nested DataFrame construction:
+
+.. code:: python
+
+   from graviti import DataFrame
+   data = [
+       {"points": [{"xmin": 1, "ymin": 3}, {"xmin": 5, "ymin": 8}]},
+       {"points": [{"xmin": 6, "ymin": 10}]},
+       {"points": [{"xmin": 1, "ymin": 3}, {"xmin": 5, "ymin": 8}, {"xmin": 1, "ymin": 9}]},
+   ]
+   df = DataFrame(data)
+
+****************
+ Read DataFrame
+****************
+
+Read data by row:
+
+.. code:: python
+
+   df.loc[0]
+
+Read data by column:
+
+.. code:: python
+
+   df[f"{COLUMN_NAME}"]
+
+Read a DataFrame cell:
+
+.. code:: python
+
+   df.loc[0][f"{COLUMN_NAME}"]
+   df[f"{COLUMN_NAME}"][0]
+
+****************
+ Edit DataFrame
+****************
+
+Modify Column Content:
+
+.. code:: python
+
+   df[f"{COLUMN_NAME}"] = [1, 4, 9]
```

Comment:
do we need to divide it to `add columns` and `modify data`? If updating data is mentioned here, do we also need to mention that series changes affect dataframe, etc

Reply:
We not support add columns and series changes affect dataframe now...